pkg/deployments/src/task.ts

```diff
@@ -414,6 +414,7 @@ export default class Task {
     return [TASKS_DIRECTORY, DEPRECATED_DIRECTORY, SCRIPTS_DIRECTORY]
       .map((dir) => fs.readdirSync(dir))
       .flat()
+      .filter((dir) => fs.lstatSync(dir).isDirectory()) // MacOS saves .DS_Store files; we only care about directories.
```

Comment:
```suggestion
      .filter((dirname) => dirname[0] != '.') // MacOS saves .DS_Store files; ignore hidden files.
```
This works for me. At this point "dir" is just a string, and lstat doesn't recognize it without the path

Reply:
Yes; that's good, and more general. Maybe base this one off #2476 and merge into it?